Gemfile

```diff
@@ -7,3 +7,4 @@ gem "minitest", "~> 5.0"
 gem "simplecov", "~> 0.17.0"
 gem "rubocop", "~> 0.80.1"
 gem "rubocop-performance", "~> 1.5.2"
+gem "webrick", "~> 1.6"
```

Comment:
Why we adding webrick?

Reply:
We're not really adding it, but making the dependency explicit. The reason for doing it is explained in the commit message. I'm happy to extract that change to a separate PR, or to rename this PR to "Improvements on jruby's CI" to better match the current changes.

Reply:
Oh I was just seeing the title `Force a newer version of `webrick`` didn't notice there was more ha!
> rename this PR to "Improvements on jruby's CI" to better match the current changes.

Maybe this 👍